config/src/config/api_config.rs

```diff
@@ -27,7 +28,6 @@ fn default_enabled() -> bool {
 impl Default for ApiConfig {
     fn default() -> ApiConfig {
         ApiConfig {
-            // disable by default until the API is ready for production
             enabled: false,
```

Comment:
set this to true will cause land blocking test failure, will have separate PR fixing it.